### ISSUE TITLE
NAR (national association of realtors) compliance

### ISSUE THREAD

**Issue by aaronamoore** [khammrich/aaron#6]
I must have My firms name on the site somewhere. It can be small and off to the side and somewhat masked but it needs to be on there as the attachment shows
[Logo Full_White On Black.pdf](https://github.com/khammrich/aaron/files/4322710/Logo.Full_White.On.Black.pdf)

**Comment by khammrich** [comment]
Yep, Exactly the one I had in mind. I am going to have it in the footer.
